cmd/gaurun/gaurun.go

```diff
@@ -107,7 +107,7 @@ func main() {
 	}
 	if gaurun.ConfGaurun.Ios.Enabled {
 		if err := gaurun.InitAPNSClient(); err != nil {
-			gaurun.LogSetupFatal(fmt.Errorf("failed to init http client"))
+			gaurun.LogSetupFatal(fmt.Errorf("failed to init http client: %v", err))
```

Comment:
In addition, would you like to change original message `failed to init http client for APNs`?